assignments/prototypes.js

```diff
@@ -32,7 +47,79 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+function Humanoid(humanoidAttributes) {
+  CharacterStats.call(this, humanoidAttributes);
+  GameObject.call(this, humanoidAttributes);
+  this.team = humanoidAttributes.team;
+  this.weapons = humanoidAttributes.weapons;
+  this.language = humanoidAttributes.language;
+} 
+Humanoid.prototype = Object.create(GameObject.prototype);
+Humanoid.prototype = Object.create(CharacterStats.prototype);
+
+Humanoid.prototype.greet = function() {
+  return `${this.name} offers a greeting in ${this.language}.`
+};
+// added prototype for stretch
+Humanoid.prototype.attack = function(attacker, defender) {
+  if (attacker.healthPoints <= 10) {
+    return attacker.destroy();
+} else if (attacker.healthPoints >= 10) {
```

Comment:
You don't actually have to add in an `attacker` parameter - you can use implicit binding with the `this` keyword instead (example: `this.destroy();` instead of `attacker.destroy();`).